cmd/login.go

```diff
@@ -191,6 +193,16 @@ func (lca loginCmdArgs) process() error {
 		}
 		// For MSI login, info success message to user.
 		glcm.Info("Login with identity succeeded.")
+	case lca.azCliCred:
```

Comment:
These aren't settable right? Is there a point to add this here?

Reply:
These are set by auto-login code.